### PR TITLE
serve _wmr.js from index.html

### PR DESCRIPTION
**Pull request by JoviDeCroock**
Resolves: https://github.com/preactjs/wmr/issues/294
Fixes: https://github.com/preactjs/wmr/issues/521

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	},
 	"scripts": {
 		"wmr": "yarn workspace wmr run",
-		"demo": "yarn workspace @examples/demo run",
+		"demo": "yarn workspace @examples/demo dev",
```
Comment (marvinhagemeister) [inline package.json]: Nit: This was probably included by accident into the changes. With the previous variant we can do stuff like `yarn demo dev` or `yarn demo build` to call the different package scripts.

**Review by marvinhagemeister — APPROVED** [review]
This is so good! Makes the dev experience so much better 🙌 💯

File: packages/wmr/src/wmr-middleware.js [hunk 2]
```diff
@@ -101,6 +106,7 @@ export default function wmrMiddleware(options) {
 			pendingChanges.add('/' + filename);
 		} else if (/\.(mjs|[tj]sx?)$/.test(filename)) {
 			if (!moduleGraph.has(filename)) {
+				onChange({ reload: true });
```
Comment (JoviDeCroock) [inline packages/wmr/src/wmr-middleware.js:109]: handles first-load when the module has not entered the graph yet and we fix the error

File: packages/wmr/src/wmr-middleware.js [hunk 3]
```diff
@@ -67,6 +67,11 @@ export default function wmrMiddleware(options) {
 		const mod = moduleGraph.get(filename);
 		if (!mod) return false;
 
+		if (mod.hasErrored) {
+			mod.hasErrored = false;
```
Comment (JoviDeCroock) [inline packages/wmr/src/wmr-middleware.js:71]: When the module is in the graph we use this

**Pull request merged**